docroot/modules/custom/sitenow_signage/sass/alertsblock.scss

```diff
@@ -0,0 +1,15 @@
+.uiowa-alerts-wrapper {
```

Comment:
It doesn't seem like we actually need to have an SCSS file being compiled here. The extra complication doesn't seem worth it for the nesting on this very small file.